cmd/driver-registrar/main.go

```diff
@@ -50,14 +55,60 @@ const (
 
 // Command line flags
 var (
-	kubeconfig        = flag.String("kubeconfig", "", "Absolute path to the kubeconfig file. Required only when running out of cluster.")
-	connectionTimeout = flag.Duration("connection-timeout", 1*time.Minute, "Timeout for waiting for CSI driver socket.")
-	csiAddress        = flag.String("csi-address", "/run/csi/socket", "Address of the CSI driver socket.")
-	showVersion       = flag.Bool("version", false, "Show version.")
-
-	version = "unknown"
+	kubeconfig              = flag.String("kubeconfig", "", "Absolute path to the kubeconfig file. Required only when running out of cluster.")
+	connectionTimeout       = flag.Duration("connection-timeout", 1*time.Minute, "Timeout for waiting for CSI driver socket.")
+	csiAddress              = flag.String("csi-address", "/run/csi/socket", "Address of the CSI driver socket.")
+	kubeletRegistrationPath = flag.String("kubelet-registration-path", "/var/lib/kubelet/plugins/csi-hostpath/csi.sock",
```

Comment:
@sbezverk 

I hit an issue on using registrar with other plugin than csi-hostpath.

I'm afraid that ```kubelet-registration-path``` should not have a default value ```"/var/lib/kubelet/plugins/csi-hostpath/csi.sock"```, as the comment says "When kubeletRegistrationPath is specified then driver-registrar ONLY acts as gRPC server which replies to registration requests initiated by kubelet's pluginswatcher infrastructure."  If there is a default value, the default value will be set to ```kubeletRegistrationPath```, even when user doesn't specify this value. Therefore, users must specify ```--kubelet-registration-path=""```. Is my understanding right?